modules/installation-mirror-repository.adoc

```diff
@@ -30,38 +30,84 @@ link:https://access.redhat.com/downloads/content/290/[{product-title} downloads
 to determine the version of {product-title} that you want to install and determine the corresponding tag on the link:https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags[Repository Tags] page.
 
 . Set the required environment variables:
```

Comment:
@jianlinliu,  will you please double-check just lines 21 to 103 of this file for me?

Reply:
For me, I did not see any difference expect removing `export`, so LGTM. 
BTW, why we not remove `export` for OCP_RELEASE line?

Reply:
I made a mistake and am fixing it. Thank you!